### PR TITLE
Basic styling for the jobs index

### PR DESCRIPTION
**Pull request by creuter**
screen shot 2016-02-22 at 2 18 30 pm

screen shot 2016-02-22 at 2 18 34 pm

### CODE REVIEW

File: web/templates/job/index.html.eex [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-      <%= job.company %>
-    </li>
-  <% end %>
-</ul>
+<section class="section">
+  <header class="section-description">
+    <h4>Job Board</h4>
+    <p>Find the best design jobs local to Raleigh, NC, include those for UI,
+    UX, graphic, and illustration&amp;designers.<p>
+    <p>Jobs are free to post, and will stay online for 30&amp;days.</p>
```
Comment (joshuaogle) [inline web/templates/job/index.html.eex]: What's up with the `&amp;`s not showing up in the template? Is that a Phoenix thing?

Comment (creuter) [inline web/templates/job/index.html.eex]: Oops, those were supposed to be `&nbsp;`. I corrected that before taking the screenshot, but forgot to push that change up.

File: web/templates/layout/app.html.eex [hunk 2]
```diff
@@ -29,7 +33,7 @@
         <%= render @view_module, @view_template, assigns %>
       </main>
```
Comment (joshuaogle) [inline web/templates/layout/app.html.eex]: :scissors:

File: web/static/css/_site-navigation.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.site-navigation {
+  li {
+    display: inline-block;
+
+    + li {
```
Comment (joshuaogle) [inline web/static/css/_site-navigation.scss:5]: That's a nice way to space items out :+1:

**Comment by joshuaogle** [comment]
Looks good! :rocket:

File: web/templates/layout/app.html.eex [hunk 4]
```diff
@@ -13,13 +13,17 @@
...
-            <li><a href="http://www.phoenixframework.org/docs">Get Started</a></li>
+          <h1 class="site-logo"><%= link("Raleigh Design", to: "/") %></h1>
+          <ul class="site-navigation">
+            <li><%= link("Member Directory", to: member_path(@conn, :index)) %></li>
+            <li><%= link("Job Board", to: job_path(@conn, :index)) %></li>
           </ul>
         </nav>
-        <span class="logo"></span>
+        <%= if assigns[:page_action] do %>
+          <%= link(assigns[:page_action][:text], to: assigns[:page_action][:url], class: "button") %>
```
Comment (creuter) [inline web/templates/layout/app.html.eex]: @drapergeek This could probably use some cleaning up

**Comment by drapergeek** [comment]
@creuter All yours

**Comment by creuter** [comment]
@drapergeek Mind giving the rest of this a once over?

**Comment by drapergeek** [comment]
LGTM :shipit:

**Pull request merged**